gwcs/selector.py

```diff
@@ -108,16 +182,25 @@ def from_vertices(cls, mask_shape, regions):
 
         Returns
         -------
-        mask : `~gwcs.selector.SelectorMask`
-            Mask to be used with `~gwcs.selector.SelectorModel`.
+        mapper : `~gwcs.selector.LabelMapperArray`
+            This models is used with `~gwcs.selector.RegionsSelector`.
+            A model which takes the same inputs as `~gwcs.selector.RegionsSelector`
+            and returns a label.
+
+
 
         Examples
-        -------_
-        mask = region.create_regions_mask_from_json((300,300), 'regions.json',
-        'region_schema.json')
+        --------
+        regions = {1: [[795, 970], [2047, 970], [2047, 999], [795, 999], [795, 970]],
+                   2: [[844, 1067], [2047, 1067], [2047, 1113], [844, 1113], [844, 1067]],
+                   3: [[654, 1029], [2047, 1029], [2047, 1078], [654, 1078], [654, 1029]],
+                   4: [[772, 990], [2047, 990], [2047, 1042], [772, 1042], [772, 990]]
+                  }
+        mapper = selector.LabelMapperArray.from_vertices((2400, 2400), regions)
```

Comment:
I wonder if it wouldn't be useful to define a dtype for vertices and make this formally an array of vertices (naturally, an array of pairs like this could be directly viewed as an array of vertices).  It would just enforce the data format more clearly I think.